<ui version="4.0" >
 <class>LatLonEditPrivate</class>
 <widget class="QWidget" name="LatLonEditPrivate" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>37</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <item>
    <widget class="KIntSpinBox" name="m_deg" >
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix" >
      <string>°</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KIntSpinBox" name="m_min" >
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix" >
      <string>′</string>
     </property>
     <property name="minimum" >
      <number>-1</number>
     </property>
     <property name="maximum" >
      <number>60</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KIntSpinBox" name="m_sec" >
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix" >
      <string>″</string>
     </property>
     <property name="minimum" >
      <number>-1</number>
     </property>
     <property name="maximum" >
      <number>60</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="m_sign" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
